docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,41 @@
+## Overview
+
+AI DIAL Analytics Realtime is a service that gathers prompt logs from [DIAL Core](https://github.com/epam/ai-dial-core) and publishes them to InfluxDB.
+The logs of prompts are the data a user provides to LLM and the responses the LLM gives to the user. A user can access analytical data through pre-set Grafana dashboards.
+
+**TODO**
+Update AI DIAL Analytics Realtime [README](https://github.com/epam/ai-dial-analytics-realtime/blob/development/README.md) with a link to this document.
+
+## Pre-requisites
+
+- [DIAL Core](https://github.com/epam/ai-dial-core)
+- [Prompt Log Collector](https://github.com/vectordotdev/vector)
+- [DIAL Analytics Realtime](https://github.com/epam/ai-dial-analytics-realtime)
+- [Influx DB](https://github.com/influxdata/influxdb)
+- [Grafana](https://github.com/grafana/grafana)
+
+### DIAL Core
+
+Follow the instructions in [AI DIAL Core](https://github.com/epam/ai-dial-core/blob/development/README.md) to learn how to configure Analytics Realtime for AI DIAL Core.
+
+**TODO**
+- Describe gflog.xml
+
+### Prompt Log Collector
```

Comment:
We should highlight the difference between prompt and system logs. This is one of the most frequently asked questions.

Reply:
done